### DISCUSSION TITLE
Background GPS updates in IOS

### DISCUSSION THREAD

**Discussion by AndyCaignie** [shinyorg/shiny#1035] [Q&A]
I am using Shiny GPS to get periodic location information. This works fine on Android, but on IOS only when the app is open.
If the user closes the app the 'background' icon disappears after a few seconds, and no positions are send anymore.
I have checked the permissions and these seem to be fine.
Anything else I could have missed?
Extra info:
- Version Xamarin forms = 5.0.0.2478
- Version Shiny.Locations = 2.5.5 (only shiny lib installed)
- IOS version = 15.5
- Visual studio = 2022 (preview)
Thx in advance,
Andy

**Comment by aritchie** [comment]
Please file issues using the issue template and provide a reproducible sample.

**Reply by AndyCaignie** [reply]
OK I will try to do this. However I also updated the GPS-Locations sample to shiny 2.5.5, and I have the same behaviour for this app: nothing is logged after the app is closed (Location access=always), and the icon disappears after a few seconds.

**Reply by aritchie** [reply]
Please follow the rules of the repo.  The source code is free, my time is not.  If you can't be bothered to submit a test case, I can't be bothered to look at things for you.  The sample sets everything, so I have no idea what settings you're using.

**Reply by AndyCaignie** [reply]
Ok fair enough. I submitted an issue with a test app
